docs/build_pipeline_controller.md

```diff
@@ -11,16 +11,23 @@ flowchart TD
 
   %% Node definitions
 predicate((PREDICATE: <br> Filter events related to <br> PipelineRuns <br> proccessed by Chains <br> that have <br> succeeded))
```

Comment:
We would now need to update the predicted to also mention that the controller also picks up failed build PLRs.

Reply:
I think you maybe missed updating this